features/admin.application-templates.v1/components/application-templates-grid.tsx

```diff
@@ -109,12 +115,20 @@ const ApplicationTemplateGrid: FunctionComponent<ApplicationTemplateGridPropsInt
      * Retrieve the filter tags from the `tags` attribute of the application templates.
      */
     const filterTags: string[] = useMemo(() => {
+
+        let tags: string[] = [];
+
+        // If the current org is a sub-organization, return only OIDC, OAuth2 and Default tags.
+        if (organizationType === OrganizationType.SUBORGANIZATION) {
+            tags = [ "OIDC", "Default", "OAuth2" ];
+
+            return tags;
+        }
```

Comment:
Shall we add some context to the comment as follows?

P.S: hope we don't need to move the `tags` variable declaration to the top.

```suggestion
        // If the current org is a sub-organization, return only OIDC, OAuth2 and Default 
        // tags as currently only standard based and M2M apps are supported in sub organization.
        // This conditional can be removed once all application templates are supported in sub organizations
        if (organizationType === OrganizationType.SUBORGANIZATION) {
            return [ "OIDC", "Default", "OAuth2" ];
        }
```